vitoriapinheiro/Delta-odonto
# Feat/backend integrations

## What I did
1. Created model and route for the banner component
2. Created model and route for the footer component
3. Integrated the frontend banner with the backend
4. Integrated the frontend footer component with the backend

## How to test
1. Install project dependencies (I recommend installing the front-end dependencies with `yarn`)
2. Run both the front and backend
3. Check if the front-end Footer and Banner both reflect the content on the backend
